Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml

```diff
@@ -153,7 +153,7 @@
   - type: GhostRole
     name: ghost-role-information-space-dragon-name
     description: ghost-role-information-space-dragon-description
-    rules: ghost-role-component-space-dragon-rules
+    rules: ghost-role-information-space-dragon-rules
```

Comment:
```suggestion
    rules: ghost-role-information-space-dragon-rules # DeltaV
```

Reply:
Is it necessary? [Upstream](https://github.com/space-wizards/space-station-14/blob/master/Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml) has it as my fix, this locale broke on a upstream merge.

Reply:
disregard, thanks for pointing this out